Project-I-Social-Card/social-card/src/components/CardComponents/Card.css

```
@@ -1 +1,40 @@
 /* All styles for your Card Components live here */
+
+    .lambdaCard {
```

Industry standard convention for css declarations is dash separated lower-case words. 

Like this:
```suggestion
    .lambda-card {
```

Javascript standard convention for variables is camelcase. 

On your next projects, try to use the industry standard conventions for css declarations.